backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/apiauditfilter/ApiAuditFilter_StepDef.java

```diff
@@ -0,0 +1,305 @@
+/*
+ * #%L
+ * de.metas.cucumber
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.cucumber.stepdefs.apiauditfilter;
+
+import com.google.common.collect.ImmutableList;
+import de.metas.audit.request.ApiRequestAudit;
+import de.metas.audit.request.ApiRequestAuditId;
+import de.metas.audit.request.ApiRequestAuditRepository;
+import de.metas.common.rest_api.common.JsonMetasfreshId;
+import de.metas.common.util.EmptyUtil;
+import de.metas.cucumber.stepdefs.DataTableUtil;
+import de.metas.cucumber.stepdefs.context.TestContext;
+import de.metas.util.Services;
+import de.metas.util.web.audit.ApiRequestReplayService;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.table.api.IADTableDAO;
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.SpringContextHolder;
+import org.compiere.model.I_AD_Issue;
+import org.compiere.model.I_AD_User;
+import org.compiere.model.I_API_Audit_Config;
+import org.compiere.model.I_API_Request_Audit;
+import org.compiere.model.I_API_Request_Audit_Log;
+import org.compiere.model.I_API_Response_Audit;
+import org.compiere.util.DB;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.adempiere.model.InterfaceWrapperHelper.saveRecord;
+import static org.assertj.core.api.Assertions.*;
+
+public class ApiAuditFilter_StepDef
+{
+	private final ApiRequestAuditRepository apiRequestAuditRepository = SpringContextHolder.instance.getBean(ApiRequestAuditRepository.class);
+	private final ApiRequestReplayService apiRequestReplayService = SpringContextHolder.instance.getBean(ApiRequestReplayService.class);
+	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+	private final IADTableDAO tableDAO = Services.get(IADTableDAO.class);
+	private final TestContext testContext;
+
+	public ApiAuditFilter_StepDef(final TestContext testContext)
+	{
+		this.testContext = testContext;
+	}
+
+	@And("all the data is reset to default")
```

Comment:
better call it something like "all the API audit data is reset"

Reply:
Sure